p2p/server.py

```diff
@@ -180,7 +181,12 @@ def _make_peer_pool(self) -> PeerPool:
 
     async def _cleanup(self) -> None:
         self.logger.info("Closing server...")
-        await asyncio.gather(self.peer_pool.cleaned_up.wait(), self.discovery.cleaned_up.wait())
+        cleanup_tasks = [self.peer_pool.cleaned_up.wait()]
```

Comment:
I tried to avoid the list and wanted to use a tuple and then just do `cleanup_tasks += (self.discovery.cleaned_up.wait())` but hell broke loose once I tried that.